configure.ac

```diff
@@ -16,7 +16,7 @@ AC_SYS_LARGEFILE
 LT_PREREQ([2.2])
 LT_INIT([win32-dll])
 
-AM_GNU_GETTEXT_VERSION([0.19])
+AM_GNU_GETTEXT_VERSION([0.18])
```

Comment:
Why this change?

Reply:
Oups, it's to build on my old ubuntu. Don't include in master branch.